Library/Homebrew/dev-cmd/livecheck.rb

```diff
@@ -90,6 +92,31 @@ def run
           end
         end
 
+        if skip_autobump?
+          autobump_files = {}
+
+          formulae_and_casks_to_check.each do |formula_or_cask|
+            tap = formula_or_cask.tap
+            next if tap.nil?
+
+            autobump_files[tap] ||= begin
+              autobump_path = tap.path/".github/autobump.txt"
+              autobump_path.exist? ? File.read(autobump_path).lines.map(&:strip) : []
+            end
+          end
+
+          formulae_and_casks_to_check = formulae_and_casks_to_check.reject do |formula_or_cask|
+            tap = formula_or_cask.tap
+            next false if tap.nil?
+
+            name = formula_or_cask.respond_to?(:token) ? formula_or_cask.token : formula_or_cask.name
+            if autobump_files[tap].include?(name)
+              odebug "Skipping #{name} as it is autobumped in #{tap}."
+              true
+            end
+          end
```

Comment:
```suggestion
          autobump_files = {}
          formulae_and_casks_to_check = formulae_and_casks_to_check.reject do |formula_or_cask|
            tap = formula_or_cask.tap
            next false if tap.nil?

            autobump_files[tap] ||= begin
              autobump_path = tap.path/".github/autobump.txt"
              autobump_path.exist? ? File.read(autobump_path).lines.map(&:strip) : []
            end

            name = formula_or_cask.respond_to?(:token) ? formula_or_cask.token : formula_or_cask.name
            if autobump_files[tap].include?(name)
              odebug "Skipping #{name} as it is autobumped in #{tap}."
              true
            end
          end
```

Unless I'm overlooking something, it seems like we could combine both these steps, so we don't have to iterate over `formulae_and_casks_to_check` twice.